docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,12 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift exposed registry
```

Comment:
Suggestion, Authentication required to push container image to OpenShift integrated Docker registry